### ISSUE TITLE
A11y: Insufficient target size for 'Close' button

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5230]
### Finding Information
- **Checkpoint ID:** 11.2.5.8/15.1
- **View/Component:** Report a Problem

### Description
The 'Close' button for screenshots does not meet the minimum 24x24 px requirement.

### Recommended Action
Increase the interactive target size to at least 24x24 px.
